projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/notification/EmailNotification.java

```diff
@@ -95,11 +70,47 @@ public void send(NotificationContent notificationContent) throws MessagingExcept
     }
   }
 
-  private boolean recipientsExist(Address[] recipients) {
-    if (recipients.length == 0) {
-      return false;
+  private void sendEmail(NotificationContent notificationContent, Address[] recipients)
+      throws MessagingException {
+    if (emailPropertiesConfiguration.isAuthEnabled()) {
+      sendAuthenticatedEmail(notificationContent, recipients);
+    } else {
+      sendUnauthenticatedEmail(notificationContent, recipients);
+    }
+  }
+
+  private void sendUnauthenticatedEmail(
+      NotificationContent notificationContent, Address[] recipients) throws MessagingException {
+    Transport transport = session.getTransport();
+    var mimeMessage = prepareMessage(notificationContent);
+    transport.send(mimeMessage, recipients);
```

Comment:
in my ide it is saying this is a static method and that you didn't need to create an instance of transport.